alexandria/core/tests/test_slugmodels.py

```diff
@@ -0,0 +1,39 @@
+"""Module to test api in a generic way."""
+
+import pytest
+from django.urls import reverse
+
+
+@pytest.mark.freeze_time("2017-05-21")
+def test_create_tags(admin_client):
+    url = reverse("tag-list")
+    # create multiple to ensure PK generation works
+    for name in ["foo", "bar", "baz"]:
```

Comment:
This would do nicely as pytest parameters. :)

Reply:
Not really. The reason we didn't discover it before was that we only created a single tag, which received the slug "" (empty string). I explicitly want multiple objects created in this run so this won't slip through.